index.html

```diff
@@ -241,5 +256,50 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <section class="form">
+      <p class="form-text">
+        <!-- I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. -->
+      </p>
```

Comment:
- [ ] Kindly make sure you uncomment this paragraph and be sure its styling sticks to the one from Figma design. I know you can do it :muscle: :muscle: